app/assets/javascripts/components/general_button.es6.jsx

```diff
@@ -0,0 +1,58 @@
+class GeneralButton extends React.Component {
+
+  static get propTypes() {
+    return {
+      action: React.PropTypes.func,
+      route: React.PropTypes.string,
+      content: React.PropTypes.string.isRequired,
+    };
+  }
+
+  static get defaultProps() {
+    return {
+      action: null,
+      route: '',
+      content: '',
+    };
+  }
+
+  get styles() {
+    return {
+      container: {
+        display: 'inline-block',
+        verticalAlgin: 'middle',
+        position: 'relative',
+        padding: '6px 12px',
+        color: 'white',
+        backgroundColor: '#68B1DE',
```

Comment:
Oh I might've said that incorrectly about css styles (good catch)! For most things we'll be going alphabetically but for css we'll have a slightly different style convention. I'll tell you guys more about it as we move along.